scripts/rabbitmq-server.bat

```diff
@@ -112,6 +112,10 @@ if "!ENV_OK!"=="false" (
     EXIT /b 78
 )
 
+if "!RABBITMQ_SCHEDULING_BIND_TYPE_FLAG!"=="" (

```

Comment:
Any reason why this didn't go into [rabbitmq-env.bat](https://github.com/rabbitmq/rabbitmq-server/blob/master/scripts/rabbitmq-env.bat)?